packages/docs/modules/page-config/runtime/usePageConfig.ts

```diff
@@ -10,6 +14,13 @@ const files = Object.entries(import.meta.glob<false, string, PageConfigJSModule>
     if (name.split('/').length > 2) { return acc }
```

Comment:
If name is `/en/page/something` then length of split by / will be bigger than 2, so code from line 19 will not be executed. I think better solution will be pass name without locale prefix to usePageConfig, instead of extending page configs dictionary with locale - it makes code logic more complex, because page configs now depends on locale...